static/app/views/stories/storyFile.tsx

```diff
@@ -1,119 +0,0 @@
-import type {ComponentProps} from 'react';
-import {Fragment} from 'react';
-import styled from '@emotion/styled';
-
-import {LinkButton} from 'sentry/components/button';
-import {CopyToClipboardButton} from 'sentry/components/copyToClipboardButton';
-import TextOverflow from 'sentry/components/textOverflow';
-import {IconGithub} from 'sentry/icons';
-import {t} from 'sentry/locale';
-import {space} from 'sentry/styles/space';
-
-import type {StoryDescriptor} from './useStoriesLoader';
-
-interface Props extends ComponentProps<'div'> {
-  story: StoryDescriptor;
-}
-
-export default function StoryFile({story, ...htmlProps}: Props) {
```

Comment:
I've removed much of this and placed the rest inside a storySourceLink component